src/game/shared/mapbase/vscript_singletons.cpp

```diff
@@ -461,6 +475,16 @@ class CScriptNetPropManager
 	// CPlayerResource::m_iHealth and CBaseEntity::m_iHealth
 	varinfo_t *GetVarInfo( CBaseEntity *pEnt, const char *szProp, int index )
 	{
+		// Copied from netpropmanager.cpp
+		for ( int i = 0; i < ARRAYSIZE( s_pszBannedNetProps ); i++ )
+		{
+			if ( V_stristr( szProp, s_pszBannedNetProps[ i ] ) != NULL)
+			{
+				// Replace any banned properties with a dummy string.
+				szProp = "Y6WP5EH4I45F2LMKSDY2";
```

Comment:
What even is the point of this? Looks like a convoluted way to return null. Just return null